client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -168,7 +168,7 @@ const SiteSetupList = ( {
 		}
 		if ( currentTaskId && currentTask && tasks.length ) {
 			const rawCurrentTask = tasks.find( ( task ) => task.id === currentTaskId );
-			if ( rawCurrentTask.isCompleted && ! currentTask.isCompleted ) {
+			if ( rawCurrentTask?.isCompleted && ! currentTask.isCompleted ) {
```

Comment:
This will fix the reported code error, but is there a bigger underlying issue? 

For this to happen there must be a `currentTask` in the state, but the props given to `SiteSetupList` must include a list of tasks that doesn't have the `currentTask`

Reply:
I can't reproduce the error without these changes if I follow the steps. If I don't set a site name, design, or domain, once I reach My Home, the `Give your site a name` task is marked as completed and I can see that:
![Captura de Pantalla 2023-07-07 a las 9 34 42](https://github.com/Automattic/wp-calypso/assets/1989914/bc07d6da-dfb4-46ca-a012-97fea195407a)

I'm not sure how I can get an undefined rawCurrentTask.